cmd/minishift/cmd/root.go

```diff
@@ -213,14 +223,11 @@ func initializeProfile() string {
 		}
 	}
 
-	// We need to initialize allinstance config as the active profile information present in it.
-	ensureAllInstanceConfigPath(constants.AllInstanceConfigPath)
 	minishiftConfig.AllInstancesConfig, err = minishiftConfig.NewAllInstancesConfig(constants.AllInstanceConfigPath)
-	if err != nil {
```

Comment:
what happened to the error?

n that case we just ignore the fact the config could not be created? This could result in issues with hostfolders which relies on the fact that the configuration file is available.

Has this been tested?

Reply:
@gbraad I have a better idea to  fix this. Yes, this is tested. But I will update it with a better solution.